pandas/core/indexes/datetimes.py

```diff
@@ -66,8 +67,13 @@ class DatetimeDelegateMixin(DatetimelikeDelegateMixin):
     # We also have a few "extra" attrs, which may or may not be raw,
     # which we we dont' want to expose in the .dt accessor.
     _extra_methods = ["to_period", "to_perioddelta", "to_julian_date", "strftime"]
-    _extra_raw_methods = ["to_pydatetime", "_local_timestamps", "_has_same_tz"]
-    _extra_raw_properties = ["_box_func", "tz", "tzinfo"]
+    _extra_raw_methods = [
```

Comment:
see my comment on the other PR, I don't find _raw_ very intuitive here (maybe _delegated?)

Reply:
ahh we *already* have _delegated_methods and so on, ok, i think we still need a better name, how would you describe these properties?

Reply:
yah, like i mentioned there im planning a significant cleanup of these